app/services/identity_linker.rb

```diff
@@ -15,10 +15,6 @@ def link_identity(**extra_attrs)
     identity
   end
 
-  def already_linked?
```

Comment:
This wasn't used anywhere -- rather than figure out how it should handle soft-deletes (where the row exists, but the thing "isn't there") I just deleted it.